src/app/series/production-calendar/production-calendar-series.component.ts

```diff
@@ -111,7 +111,8 @@ export class ProductionCalendarSeriesComponent implements OnInit {
       this.series.doc.followItems('prx:stories', {
         per: this.PER_SERIES, // don't use per/count for now to work around halcache sticking on an extra Bearer token
         filters,
-        sorts: 'published_released_at: asc'
+        sorts: 'published_released_at: asc',
+        zoom: 'prx:audio-versions'
```

Comment:
zoom to just the audio-version for the version label. I couldn't easily get the story audio version to not load the version template and audio files as well, but this seems better than embedding/loading everything for each story. Audio versions loads related by default, and the episode form depends on it.